test/e2e/deployimage/plugin_cluster_test.go

```diff
@@ -263,6 +266,24 @@ func Run(kbc *utils.TestContext) {
 	}
 	EventuallyWithOffset(1, getMemcachedPodStatus, time.Minute, time.Second).Should(Succeed())
 
+	By("validating that the status of the custom resource created is updated or not")
+	var status string
+	var customResourceConditions metav1.Condition
+	//nolint:lll
+	getStatus := func() error {
+		status, err = kbc.Kubectl.Get(true, strings.ToLower(kbc.Kind),
```

Comment:
Can we do as above ?

```
status, err := kbc.Kubectl.Get(true, "pods", "-l",
			fmt.Sprintf("app.kubernetes.io/name=%s", strings.ToLower(kbc.Kind)),
			"-o", "jsonpath={.items[*].status}",
		)
		ExpectWithOffset(2, err).NotTo(HaveOccurred())
```

Then, if has no error we can unmarshal the result and check.
Also, what is returned by {`.status.conditions}`?